pkg/controller/manager/manager_controller_test.go

```diff
@@ -1341,6 +1341,180 @@ var _ = Describe("Manager controller tests", func() {
 				Expect(kerror.IsNotFound(test.GetResource(c, &tenantBRoutes))).Should(BeTrue())
 			})
 		})
+
+		Context("TSL route misconfiguration", func() {
+			var certificateManager certificatemanager.CertificateManager
+			BeforeEach(func() {
+				mockStatus.On("AddDaemonsets", mock.Anything).Return()
+				mockStatus.On("AddDeployments", mock.Anything).Return()
+				mockStatus.On("AddStatefulSets", mock.Anything).Return()
+				mockStatus.On("AddCronJobs", mock.Anything)
+				mockStatus.On("IsAvailable").Return(true)
+				mockStatus.On("OnCRFound").Return()
+				mockStatus.On("ClearDegraded")
+				mockStatus.On("RemoveCertificateSigningRequests", mock.Anything)
+				mockStatus.On("ReadyToMonitor")
+				mockStatus.On("SetMetaData", mock.Anything).Return()
+
+				// Mark that watches were successful.
+				r.licenseAPIReady.MarkAsReady()
+				r.tierWatchReady.MarkAsReady()
+			})
+			It("single-tenant", func() {
```

Comment:
Nit, the description of the test with it should flow with "It <given descritpion>", so this reads as "It single-tenant".

So this should probably be something like `It("creates the manager deployment and degrades the tigera status for a single tenant")`. Could also be wrapped in another `Context` or `When` saying if it's single tenant or multi tenant.